sql/item_mods.sql

```diff
@@ -33057,6 +33070,9 @@ INSERT INTO `item_mods` VALUES (17064,10,2); -- VIT: 2
 -- Platinum Rod
 INSERT INTO `item_mods` VALUES (17065,2,18); -- HP: 18
 
+-- Molva Maul (17066)
+-- TODO: Latent effect: "Randgrith"
```

Comment:
No need for comments in the mods table for things that are latent.

Reply:
Honestly, this was between other additional effects items in my wip document, so it sort of got thrown in there. Removed.